src/main.cpp

```diff
@@ -1143,6 +1143,23 @@ std::string FormatStateMessage(const CValidationState &state)
         state.GetDebugMessage().empty() ? "" : ", " + state.GetDebugMessage(), state.GetRejectCode());
 }
 
+#ifdef BITCOIN_CASH
+static bool IsCashHFEnabled(const Config &config, int64_t nMedianTimePast)
+{
+    return nMedianTimePast >= config.GetChainParams().GetConsensus().cashHardForkActivationTime;
+}
+
+bool IsCashHFEnabled(const Config &config, const CBlockIndex *pindexPrev)
+{
+    if (pindexPrev == nullptr)
+    {
+        return false;
+    }
+
+    return IsCashHFEnabled(config, pindexPrev->GetMedianTimePast());
+}
+#endif
+
```

Comment:
Can we get started with isolation by putting these functions in a bitcoincash.cpp and .h file?

Reply:
done.